sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -329,33 +343,77 @@ class SessionCatalog(
   // ----------------------------------------------
 
   /**
-   * Create a temporary table.
+   * Create a local temporary view.
    */
   def createTempView(
       name: String,
-      tableDefinition: LogicalPlan,
+      viewDefinition: LogicalPlan,
       overrideIfExists: Boolean): Unit = synchronized {
-    val table = formatTableName(name)
-    if (tempTables.contains(table) && !overrideIfExists) {
+    val viewName = formatTableName(name)
+    if (tempTables.contains(viewName) && !overrideIfExists) {
       throw new TempTableAlreadyExistsException(name)
```

Comment:
Maybe good to avoid of having this kind of changes in this PR? We can submit a follow-up one to make the variable name consistent.